docs/usage/serve_files.rst

```diff
@@ -0,0 +1,39 @@
+..
+    This file is part of Invenio.
+    Copyright (C) 2015-2019 CERN.
+
+    Invenio is free software; you can redistribute it and/or modify it
+    under the terms of the MIT License; see LICENSE file for more details.
+
+Serve Files
+-----------
+Once we have created our bucket and uploaded a file, it is time to access it.
+You can simply access the desired file by firing a request in the url.
+By default, the latest version will be downloaded. In case we want a different version, we can specify which one in the query parameters.
+
+Download the latest version of the file:
+
+.. code-block:: console
+
+   $ curl -i http://localhost:5000/files/$B/my_file.txt
+
+Download a specific version of the file:
+
+.. code-block:: console
+
+   $ curl -i http://localhost:5000/files/$B/my_file.txt?versionId=<version_id>
+
+**NOTE:** By default, the file contents are displayed in the browser.
+Should you want to download the file, add the :code:`download` keyword in the query parameters.
+
```

Comment:
`
Retrieve Files
-------------

Once we have created our bucket and uploaded a file, it is possible to retrieve it with a GET request.
By default, the latest version will be downloaded.

ex

If you want to retrieve a specific version of the file, you can use the versionId query parameter:

 **NOTE:** By default, the file is server with the header 'Content-Disposition': 'inline', meaning that the browser will try to preview it if possible. If you want to trigger a download, you can use the extra query parameter `download`, which will change the 'Content-Disposition' header to 'attachment'
`